cores/arduino/SERCOM.cpp

```diff
@@ -652,20 +652,16 @@ void SERCOM::initClockNVIC( void )
     clockId = GCM_SERCOM3_CORE;
     IdNvic = SERCOM3_IRQn;
   }
-#if defined(SERCOM4)
```

Comment:
We shouldn't do it this way. Instead we need to look at defining SERCOM4 and 5 -- they have to be defined, or this code wouldn't compile.